src/test/java/org/apache/drillui/test/framework/steps/webui/ProfilesSteps.java

```diff
@@ -0,0 +1,10 @@
+package org.apache.drillui.test.framework.steps.webui;
+
+import org.apache.drillui.test.framework.pages.BasePage;
+import org.apache.drillui.test.framework.pages.ProfilesPage;
+
+public class ProfilesSteps {
+public boolean isQueryProfileINList(String queryProfile){
+    return BasePage.getPage(ProfilesPage.class).isQueryProfileINList(queryProfile);
+}
```

Comment:
Please setup your IDEA and align your code.

Reply:
Done